integrationtests/IntegrationTests.Shared.Tests/Features/Login/LoginViewModelTests.cs

```diff
@@ -65,115 +169,51 @@ public async Task LoginButton_IsEnabled_WhenUserNameAndPassword_IsNotEmptyAsync(
                 .WithUserName(userName)
                 .WithPassword(password);
 
-            var result = await sut.Login.CanExecute.FirstAsync();
-            result.Should().BeTrue();
+            (await sut.Login.CanExecute.FirstAsync()).ShouldBe(true);
         }
 
         /// <summary>
-        /// Checks to make sure that the login button is disabled when not logging in.
-        /// </summary>
-        /// <returns>A task to monitor the progress.</returns>
-        [Fact]
-        public async Task CancelButton_IsDisabled_WhenNot_LoggingIn()
-        {
-            LoginViewModel sut = new LoginViewModelBuilder();
-
-            var result = await sut.Cancel.CanExecute.FirstAsync();
-            result.Should().BeFalse();
-        }
-
-        /// <summary>
-        /// Checks to make sure that the cancel command actually cancels a login attempt.
-        /// </summary>
-        /// <returns>A task to monitor the progress.</returns>
-        [Fact]
-        public async Task CancelButton_Cancels_Login()
-        {
-            await new TestScheduler().With(async sched =>
-            {
-                LoginViewModel sut = new LoginViewModelBuilder()
-                    .WithScheduler(sched)
-                    .WithUserName("coolusername")
-                    .WithPassword("excellentpassword");
-
-                sut.Login.Subscribe(x => x.Should().BeTrue());
-
-                sched.AdvanceByMs(TimeSpan.FromSeconds(1).Milliseconds);
-
-                var result = await sut.Cancel.CanExecute.FirstAsync();
-                result.Should().BeTrue();
-            }).ConfigureAwait(false);
-        }
-
-        /// <summary>
-        /// Checks to make sure that the cancel button is available within two seconds.
+        /// Checks to make sure the user can login with a correct password.
         /// </summary>
-        /// <returns>A task to monitor the progress.</returns>
         [Fact]
-        public async Task CancelButton_IsAvailableUntil_TwoSeconds()
+        public void User_CanLogin_WithCorrect_Password()
         {
-            new TestScheduler().With(sched =>
-            {
-                LoginViewModel sut = new LoginViewModelBuilder()
-                    .WithScheduler(sched)
-                    .WithUserName("coolusername")
-                    .WithPassword("excellentpassword");
-
-                sut.Login.Execute().Subscribe();
-
-                sut.Cancel.CanExecute
-                    .FirstAsync().Wait()
-                    .Should().BeFalse();
-
-                // 50ms
-                sched.AdvanceByMs(50);
-
-                sut.Cancel.CanExecute
-                    .FirstAsync().Wait()
-                    .Should().BeTrue();
+            var scheduler = new TestScheduler();
+            LoginViewModel sut = new LoginViewModelBuilder()
+                .WithScheduler(scheduler)
+                .WithUserName("coolusername")
+                .WithPassword("Mr. Goodbytes");
```

Comment:
Why use builders as separate classes instead of newing up the view model directly? 
Wouldn't it be more concise and easy if it was:
```cs
var vm = new LoginViewModel(scheduler) 
{ 
    UserName = "coolusername", 
    Password = "Mr. Goodbytes" 
};
```

Reply:
Probably outside the scope of this review since it's used everywhere. Worth putting a issue in though